src/alt-gosling-model/alt-structure/chart-types.ts

```diff
@@ -44,9 +45,18 @@ export function determineSpecialCases(altTrack: AltTrackSingle | AltTrackOverlai
     if (_mark === 'rect' && _genomicEncodings.includes('x') && _genomicEncodings.includes('xe') && _quantitativeEncodings.includes('color')) {
         return `${layoutDesc}heat map`;
     }
-    if (_mark === 'rect' && _genomicEncodings.includes('x') && _genomicEncodings.includes('xe') && _nominalEncodings.includes('color')) {
+    if (_mark === 'rect' && _dataType !== 'beddb' && _genomicEncodings.includes('x') && _genomicEncodings.includes('xe') && _nominalEncodings.includes('color')) {
```

Comment:
I think it will be more precise if we specify the possible data types. For example, bam files also contain genomic range annotations, but they do not store data for drawing ideograms.

```suggestion
    if (_mark === 'rect' && ['csv', 'json'].includes(_dataType) && _genomicEncodings.includes('x') && _genomicEncodings.includes('xe') && _nominalEncodings.includes('color')) {
```